packages/cli/src/ensure-configuration.ts

```diff
@@ -1,15 +1,11 @@
 import { ProjectOptions } from '@bigtest/project';
+import { MainError } from '@effection/node';
 import fs from 'fs';
 
 export const ensureConfiguration = (config: ProjectOptions): void => {
   if (typeof config.tsconfig !== 'undefined' && fs.existsSync(config.tsconfig) === false) {
-    let errorMessage = `The \`tsconfig\` option of bigtest.json (\`${config.tsconfig}\`) is invalid.`;
+    let message = `The \`tsconfig\` option of bigtest.json (\`${config.tsconfig}\`) is invalid.`;
```

Comment:
small nit, but I believe we support loading different configuration files, so we can't assume that it will be named `bigtest.json`. If possible, we should point to the exact file.